setup.py

```diff
@@ -90,7 +90,5 @@
                               'Operating System :: POSIX',
                               'Operating System :: Unix',
                               'Operating System :: MacOS',
-                              'Programming Language :: Python :: 3',
```

Comment:
Is there a reason why you dropped the Python :: 3 tag?

Reply:
I guess I should have asked to be sure. I presumed that it covers as a <= check.
I'll put it back?

Reply:
These are actually not checks, but tags displayed on pypi.org.